daemon/daemon_no_aufs.go

```diff
@@ -1,4 +1,4 @@
-// +build exclude_graphdriver_aufs,linux
+// +build exclude_graphdriver_aufs,linux !linux
```

Comment:
daemon_no_aufs and daemon_aufs.go aren't currently being built on Windows with the tags as they are as the only call to migrateIfAufs is only made from daemon_unix.go which isn't compiled on Windows. Adding a !linux tag in daemon_no_aufs.go will mean daemon_no_aufs.go will be compiled on Windows which isn't necessary. I just verified that by deliberately putting compile errors in both daemon_no_aufs and daemon_aufs.go and successfully compiling the Windows daemon. I then added the tag above and sure enough, it tripped on the compile error I added to daemon_no_aufs. So I really don't believe this change is necessary here :)